iohk-monitoring/src/Cardano/BM/Data/Configuration.lhs

```diff
@@ -17,10 +17,13 @@ module Cardano.BM.Data.Configuration
   (
     Representation (..)
   , Port
-  , parseRepresentation
```

Comment:
external code should only call 'readRepresentation'

Reply:
Why? I have the a`ByteString` and would like to parse its contents without writing it to a file first.